From 5f9872295f6ad29e86832ce56b08d0224600a32c Mon Sep 17 00:00:00 2001
From: Eric Yang <eric.yang2@amd.com>
Date: Wed, 13 Jul 2016 17:57:51 -0400
Subject: [PATCH 0645/1722] drm/amd/dal: do not lower display clock

Currently the display clock is lowered too quickly, since many
register programming are double buffered with vupdate, but
displayclock change is not. Need to revisit how to do this.

Signed-off-by: Eric Yang <eric.yang2@amd.com>
Acked-by: Harry Wentland <harry.wentland@amd.com>
Signed-off-by: Alex Deucher <alexander.deucher@amd.com>
Signed-off-by: Kalyan Alle <kalyan.alle@amd.com>
---
 drivers/gpu/drm/amd/dal/dc/core/dc.c | 11 ++---------
 1 file changed, 2 insertions(+), 9 deletions(-)

diff --git a/drivers/gpu/drm/amd/dal/dc/core/dc.c b/drivers/gpu/drm/amd/dal/dc/core/dc.c
index f21f738..7df5af1 100644
--- a/drivers/gpu/drm/amd/dal/dc/core/dc.c
+++ b/drivers/gpu/drm/amd/dal/dc/core/dc.c
@@ -969,15 +969,8 @@ bool dc_commit_surfaces_to_target(
 				&core_dc->current_context->res_ctx);
 
         core_dc->hwss.apply_ctx_to_surface(core_dc, context);
-
-        /* TODO: decouple wm programming and display clock and unhack this condition*/
-
-	/* Lower display clock if necessary */
-        if (prev_disp_clk > context->bw_results.dispclk_khz) {
-                core_dc->hwss.set_display_clock(context);
-		pplib_apply_display_requirements(core_dc, context,
-						&context->pp_display_cfg);
-	}
+ 
+        /* TODO: Implement lowering display clock */
 
 	resource_validate_ctx_destruct(core_dc->current_context);
 	dm_free(core_dc->current_context);
-- 
2.7.4

